### ISSUE TITLE
NTR, pulmo: Elevated RV/TLC ratio

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6446]
**Preferred term label:** Elevated RV/TLC ratio

**Synonyms**

**Definition (free text, please give PubMed ID)**
Sign for Air trapping and Hyperinflation. Comment: Hyperinflated lungs occur when air gets trapped in the lungs and causes them to overinflate. Hyperinflated lungs can be caused by blockages in the air passages or by air sacs that are less elastic, which interferes with the expulsion of air from the lungs

**Parent term (use hpo.jax.org/app)**
Abnormal spirometry test HP:0032340

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU: chrapp, @mgriese, @kknoflach

**Comment by pnrobinson** [comment]
@GrieseAG  according to https://pubmed.ncbi.nlm.nih.gov/12934788/ RV/TLC cannot be measured by spirometry, therefore can we discuss the parent term?
Here is a proposed extended definition:
def: An increase in the ratio of residual volume (RV) to total lung capacity (TLC) on pulmonary function testing. RV is the amount of air remaining aftermaximal expiration and TLC is the total amount of air in theungs at full inspiration. These volumes cannot be determined by spirometry, but can be measured by inert gas dilution, nitrogen washout, and (3) body plethysmography. An elevated RV/TLC can be interpreted as a sign of air trapping and hyperinflation. PMID:12934788

**Comment by pnrobinson** [comment]
done
